apps/site/pages/en/learn/modules/package-configuration.md

```diff
@@ -0,0 +1,563 @@
+
+- CJS source and distribution with good/specific `module.exports`
+- ESM source and distribution
+  - If your source-code does _not_ contain any top-level `await`, your ESM likely can be `required()`, meaning it can be consumed by a project using CJS. See [`require()` an ES Module](https://nodejs.org/api/modules.html#loading-ecmascript-modules-using-require).
+
+| You as a package author write     | Consumers of your package write their code in                    | Your options                                                                                                                                                 |
+| :-------------------------------- | :--------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| CJS source code using `require()` | CJS: consumers `require()` your package                          | [CJS source and distribution](#cjs-source-and-distribution)                                                                                                  |
+| CJS source code using `require()` | ESM: consumers `import` your package                             | [CJS source and only ESM distribution](#cjs-source-and-only-esm-distribution)                                                                                |
+| CJS source code using `require()` | CJS & ESM: consumers either `require()` or `import` your package | [CJS source and both CJS & ESM distribution](#cjs-source-and-both-cjs-amp-esm-distribution)                                                                  |
+| ESM source code using `import`    | CJS: consumers `require()` your package                          | [ESM source with only CJS distribution](#esm-source-with-only-cjs-distribution)<br />[ESM source and distribution](#esm-source-and-distribution)             |
+| ESM source code using `import`    | ESM: consumers `import` your package                             | [ESM source and distribution](#esm-source-and-distribution)                                                                                                  |
+| ESM: source code uses `import`    | CJS & ESM: consumers either `require()` or `import` your package | [ESM source and both CJS & ESM distribution](#esm-source-and-both-cjs-amp-esm-distribution)<br />[ESM source and distribution](#esm-source-and-distribution) |
+
+### CJS source and distribution
+
+This the "Rum & Coke" of packages: pretty difficult to mess up. You _technially_ may not need _any_ package configuration beyond [`"name"`](https://nodejs.org/api/packages.html#name). But the less arcane, the better: Essentially just declare the package’s exports via the `"exports"` field/field-set.
```

Comment:
For official tutorials, references to alcoholic drinks seem inappropriate. Also as a non-drinker I feel these references just add an extra layer of confusion for me when I read it...

Reply:
They could be virgin drinks—nowhere does it mention alcohol :)

> Also as a non-drinker I feel these references just add an extra layer of confusion for me

I don't understand why. These are not at all keystones in explanations—merely passing humour that do perhaps also provide a tiny bit of insight.

The inclusion of these is to add some humour to an otherwise dry and boring topic. A lot of readers specifically cited enjoying these. Just because some people don't enjoy them doesn't mean others shouldn't get the chance.

Perhaps sobriety is a reason, but this is hardly someone's only exposure to it. And again, there is no mention of actual alcohol.

We can remove them, but I think it would be much to the detriment.

Reply:
Let me think about how to fix it without losing the humour.